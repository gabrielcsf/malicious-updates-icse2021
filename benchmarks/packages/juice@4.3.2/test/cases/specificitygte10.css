div.a.b.c.d.e.f.g.h.i.j {
  padding: 10px;
}
div div.a {
  padding-top: 0;
}