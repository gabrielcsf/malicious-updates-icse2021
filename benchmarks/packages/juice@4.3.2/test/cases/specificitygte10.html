<html><body>
<div>
  <div class="a b c d e f g h i j k l m n o"></div>
</div>
</body></html>
