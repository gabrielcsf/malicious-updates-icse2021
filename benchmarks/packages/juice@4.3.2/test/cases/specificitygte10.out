<html><body>
<div>
  <div class="a b c d e f g h i j k l m n o" style="padding-top: 0; padding: 10px;"></div>
</div>
</body></html>
